core/src/main/java/org/apache/brooklyn/location/ssh/SshMachineLocation.java

```diff
@@ -20,17 +20,7 @@
 
 import static org.apache.brooklyn.util.groovy.GroovyJavaMethods.truth;
 
-import java.io.Closeable;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.io.PipedInputStream;
-import java.io.PipedOutputStream;
-import java.io.Reader;
-import java.io.StringReader;
+import java.io.*;
```

Comment:
Also recommend that you configure your IDE to not use `*` for imports. The convention in Brooklyn is to be explicit about the import classes.